master/buildbot/steps/artifact.py

```diff
@@ -298,6 +362,13 @@ def rsyncWithRetry(step, origin, destination, port=None):
 
     return retryCommandLinuxOS(rsync_command)
 
+def mkDir(step):
+    if _isWindowsSlave(step):
+        return r'C:\cygwin64\bin\mkdir.exe'
```

Comment:
I certainly think it's a bad idea to put this in like this, this is 100% unity specific and even then it's 100% agent specific, please use mkdir.exe and have people put it in their PATHs instead